src/components/main-view/main-view.jsx

```diff
@@ -33,7 +41,6 @@ export const MainView = () => {
           Title: movie.Title,
```

Comment:
you could[ destructure](https://www.w3schools.com/react/react_es6_destructuring.asp) out everything you need from movie: 

```
const moviesFromApi = movies.map(({
  _id,
  Title,
  ImagePath,
  Description,
  Genre: { Name: GenreName },
  Director: { Name: DirectorName, Bio: DirectorBio, Birth: DirectorBirth },
  Featured
}) => ({
  _id,
  Title,
  ImagePath,
  Description,
  Genre: { Name: GenreName },
  Director: { Name: DirectorName, Bio: DirectorBio, Birth: DirectorBirth },
  Featured
}));
```


This approach leads to cleaner and more readable code, especially when dealing with objects that have multiple properties.